packages/happn-cluster/README.md

```diff
@@ -210,105 +182,79 @@ Configure proxy's request-side socket's idle timeout. Default 20 minutes.
-
-#### config.stabiliseTimeout
-
-Defines how long to wait for this starting node to become fully connected (stabilised) before giving up and stopping the node. In all known cases a starting node will either reach stability or fail explicitly with an error. This is a failsafe (for unknown cases) to prevent endlessly awaiting stability where it would be better to stop and try joining the cluster again.
-
-**Note that this acts in opposition to `minimumPeers` - A starting node awaiting minimum peers will still time out.**
-
-### Membership Sub-Config
-
 #### config.clusterName
 
 Every member of the cluster should have the same configured `clusterName`.
-The name is limited to characters acceptable in happn paths, namely '\_*-', numbers and letters.
+The name is limited to characters acceptable in happn paths, namely '\_\*-', numbers and letters.
 Joining members with a different clusterName will be ignored by the orchestrator.
 
-#### config.seed
-
-Boolean flag sets this member as the cluster seed member. If `true` this member will not terminate
-upon failing to join any other cluster members and can therefore enter a cluster successfully as
-the first member.
-
-Each other member should include the seed member, among others, in their **config.hosts**
-list of hosts to join when starting.
-
-#### config.seedWait
-
-Members that are not the seed member pause this long before starting. This allows for a booting host that
-contains multiple cluster member instances all starting concurrently where one is the seed member. By waiting, the seed member will be up and running before the others attempt to join it in the cluster.
-
-#### config.randomWait
-
-Members that are not the seed member pause a random amount of milliseconds from this range before starting. This is to alleviate the problem of SWIM not handling large numbers of concurrent cluster joins.
-
-Be sure to set this value to less than the stabiliseTimeout.
-
-#### config.[host, port]
-
-The host and port on which this member's SWIM service should listen. Host should be an actual ip address
-or hostname, not '0.0.0.0'. It can also be specified using [dface](https://github.com/happner/dface) spec.
-
-**Important: The membership service protocol is currently insecure. These ports need to be protected in a private cluster.** [issues/1](https://github.com/happner/happn-cluster/issues/1)
-
-#### config.hosts
+#### config.serviceName
 
-The list of initial cluster members via which this member joins the cluster. This should include the
-seed member and a selection of other members likely to be online.
+Every member of the cluster should have a serviceName.
+Cluster will not be stable until a minimum number of each service by name, as defined in `config.cluster`
+have joined the cluster. The serviceName is not unique by node. Multiple nodes can be of the same service, 
+and any requests to components on those nodes will round-robin between the nodes which have that component.
```

Comment:
Is there a test for this?

Reply:
Good question. There are tests that prove round-robbining works between compomemts of the same name. Since the serviceName is a very high-level abstraction, it should not change the behaviour at the component level. I will write a test that proves this.

Reply:
Tested